### PR TITLE
Use logging instead of print statements

### PR DESCRIPTION
**Pull request by dvf**
Hey folks,

There are a few `print()` statements in the message module that make it very difficult to see what's happening when running in a server environment.

This change uses the Python `logging` module instead which can be toggled to DEBUG mode to produce the same output if needed.

### CODE REVIEW

File: silasdk/message.py [hunk 1]
```diff
@@ -95,8 +96,7 @@ def createMessage(self, payload, msg_type):
         pass
 
     inpt = cull_null_values(inpt, payload)
-    if (self.debug):
-        print(inpt)
+    logger.debug(inpt)
```
Comment (johndburk) [inline silasdk/message.py:99]: In order to respect `self.debug` regardless of the configured logging level, you should get / optionally set / reset the logging levels:
```
log_level = logger.getEffectiveLevel()
if self.debug:
    logger.setLevel(logging.DEBUG)
logger.debug(inpt)
logger.setLevel(log_level)
```

File: silasdk/message.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def postRequest(app: App, path: str, msg_type: str, payload: dict, key: Optional
         key :user_private_key
     """
     data = createMessage(app, payload, msg_type)
-    print(data)
+    logger.debug(data)
```
Comment (johndburk) [inline silasdk/message.py:112]: No access to `self` here, so no way to determine desired logging level.  But I'm in agreement here that suppressing this message by default is appropriate behavior, and the SDK consumer can set the default level to DEBUG as desired.

File: README.md [hunk 3]
```diff
@@ -5,4 +5,14 @@
 A guide to the installation and configuration of this SDK can be found here: [https://docs.silamoney.com/docs/python-sdk](https://docs.silamoney.com/docs/python-sdk)
 
 ### Endpoint Methods
-Example code for each SDK endpoint method is found in the associated endpoint's section in our docs, which can be found here: [https://docs.silamoney.com/docs/get-started](https://docs.silamoney.com/docs/get-started)
\ No newline at end of file
+Example code for each SDK endpoint method is found in the associated endpoint's section in our docs, which can be found here: [https://docs.silamoney.com/docs/get-started](https://docs.silamoney.com/docs/get-started)
+
+### Debug Logging
+
+If you'd like to set additional debug logging, then make sure to set the logging level when you run your code:
```
Comment (johndburk) [inline README.md:12]: Add a note here mentioning that if you don't call `logging.basicConfig`, the logging level is set to `WARNING` by default, and so the `logger.debug()` messages will not be logged to the console.

**Review by johndburk — CHANGES_REQUESTED** [review]

**Comment by dvf** [comment]
Feel free to make the changes :+1: 

The print statements were probably left there while debugging. We don't have use for them.

File: silasdk/message.py (hunk 1 above)
Comment (dvf) [reply silasdk/message.py:99]: It's best practice to set log levels globally regardless of what a specific library's mode is. 

Also, just to clarify: I don't know the reasons behind logging here, I'm just replacing the existing `print` statements.

Comment (johndburk) [reply silasdk/message.py:99]: > It's best practice to set log levels globally regardless of what a specific library's mode is.

Unless you want debug logging from just a single module or class, and don't want to see the heinous amount of debug logging that boto3 or other misc packages can spew at times :-)

**Comment by dvf** [comment]
Looks like the print statements were removed. Thanks!

**Pull request closed without merging**